packages/tqdm/tqdm.0.1/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tqdm/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tqdm"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tqdm.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-j" jobs "-p" name]]
+
+depends: [
+  "base" {>= "v0.12.0" & < "v0.13"}
+  "dune" {>= "1.3.0" & < "2.0"}
```

Comment:
Looks like it requires dune >2

```
#=== ERROR while compiling tqdm.0.1 ===========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | pinned(https://github.com/LaurentMazare/ocaml-tqdm/archive/0.1.tar.gz)
# path                 ~/.opam/4.09/.opam-switch/build/tqdm.0.1
# command              ~/.opam/4.09/bin/dune build -j 2 -p tqdm
# exit-code            1
# env-file             ~/.opam/log/tqdm-23-a955d2.env
# output-file          ~/.opam/log/tqdm-23-a955d2.out
### output ###
# File "dune-project", line 1, characters 11-14:
# 1 | (lang dune 2.0)
#                ^^^
# Error: Version 2.0 of dune is not supported.
# Supported versions:
# - 0.0
# - 1.0 to 1.11
```

Reply:
Ah right, sorry about this and thanks for pointing it out. I'll have a go at fixing this.

Reply:
Looks like the issue was caused by my local install using a pinned dev version of dune pre-2.0 hence the 2.0 in `dune-project` while the dune files were still using the 1.1 syntax.